pages/email.tsx

```diff
@@ -32,14 +32,21 @@ export default function Email() {
   const [isIdle, setIsIdle] = useState(false)
 
   const handleOnIdleTimerIdle = useCallback(() => {
+    deleteCookie('agreed-to-email-esrf-terms')
+    router.push('/landing')
+  }, [router])
+
+  const handleOnIdleTimerPrompt = useCallback(() => {
     setIsIdle(true)
     setModalOpen(true)
   }, [])
 
   const { reset: resetIdleTimer } = useIdleTimer({
     onIdle: handleOnIdleTimerIdle,
-    //15 minute timeout
-    timeout: 15 * 60 * 1000,
+    onPrompt: handleOnIdleTimerPrompt,
+    //10 minute timeout followed by a 5 minute prompt timeout
+    timeout: 15 * 40 * 1000,
```

Comment:
The timeout and promptTimeout calculations are confusing.

This is less configusing, imo:

```js
timeout: 10 * 60 * 1000, // 10 mins
promptTimeout: 5 * 60 * 1000,  // 5 mins
```

Total mins = {mins} * {seconds} * {milliseconds}
1 min = 60 seconds * 1000 milliseconds

Please fix here and in status page.

Reply:
fixed